docs/source/host/server-setup/linux-server.md

```diff
@@ -66,14 +66,58 @@ The installation package is configured to permit multiple domains to run on a si
 
 where <code>name</code> is a word used to name and manage the domain and <code>base-port</code> must be the the first of a range of four contiguous port numbers not overlapping with any other use on the system.
 
-Assuming you created a new server with the name **second-bite**, this would setup the following:
- - Environment variables in <code>/etc/opt/vircadia/**second-bite**.conf</code>
- - Content stored in <code>/var/lib/vircadia/**second-bite**</code>
- - Services launched as <code>vircadia-domain-server@**second-bite**.service</code>, <code>vircadia-assignment-client@**second-bite**.service</code>, and <code>vircadia-server@**second-bite**.target</code>
+Assuming you created a new server with the name **my-server-two**, this would setup the following:
+ - Environment variables in <code>/etc/opt/vircadia/**my-server-two**.conf</code>
+ - Content stored in <code>/var/lib/vircadia/**my-server-two**</code>
+ - Services launched as <code>vircadia-domain-server@**my-server-two**.service</code>, <code>vircadia-assignment-client@**my-server-two**.service</code>, and <code>vircadia-server@**my-server-two**.target</code>
+
+## Deleting a Vircadia Server
+
+Uninstall the package.
+
+```sh
+# Ubuntu
+sudo apt-get purge vircadia-server
```

Comment:
Keep in mind that this also removes all configuration files as oppose to `apt-get remove`. As pretty much all (if not all) package managers don't remove configuration files by standard, we should leave a little note here that this command does delete everything.